include/wx/mdi.h

```diff
@@ -142,8 +142,13 @@ class WXDLLIMPEXP_CORE wxMDIParentFrameBase : public wxFrame
 // ----------------------------------------------------------------------------
 // wxMDIChildFrameBase: child frame managed by wxMDIParentFrame
 // ----------------------------------------------------------------------------
+#ifdef __WXQT__
+using wxMDIChildFrameBaseBase = wxFrameBase;
+#else
+using wxMDIChildFrameBaseBase = wxFrame;
+#endif
```

Comment:
This is unfortunate because, as the changes to the sample show, almost all code using MDI will probably need to be changed to compile with wxQt. I wonder why does it need this when the other ports, even those using TDI, can live with inheriting from `wxFrame` here?